Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -340,9 +340,9 @@ inline Gfx::ScalingMode to_gfx_scaling_mode(CSS::ImageRendering css_value, Gfx::
             return Gfx::ScalingMode::BoxSampling;
         return Gfx::ScalingMode::BilinearBlend;
     case CSS::ImageRendering::CrispEdges:
-        return Gfx::ScalingMode::NearestNeighbor;
-    case CSS::ImageRendering::Pixelated:
         return Gfx::ScalingMode::SmoothPixels;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering CrispEdges should go to NearestNeighbor. Pixelated could do something more complicated, but mapping it to NearestNeighbor seems alright until we have the specced behavior. (SmoothPixels doesn't look like it's what Pixelate's description should do.)

I.e. this could change to map Pixelated to NearestNeighbor, and that's probably a slight progression.

Reply:
I agree, see also my comment on the issue: https://github.com/LadybirdBrowser/ladybird/issues/1170#issuecomment-2307749394